lib/string_segmenter.rb

```diff
@@ -1,2 +1,10 @@
+require 'dictionary'
 def segment_string(str)
-end
\ No newline at end of file
+  words = []
+  str.downcase.delete('^a-z ')
+  if str.include dictionary
+    then words(//) << str
```

Comment:
Do you mean `words(//)` here? What is the `(//)` for? What happens if you remove it?